src/core/utils/bounding-box.ts

```diff
@@ -189,10 +189,10 @@ function traverseAndCollectTrackInfo(
 
         if (spec.orientation === 'vertical') {
             // This is a vertical view, so use the largest `height` of the tracks for this view.
-            cumHeight = Math.max(...tracks.map(d => d.height));
+            cumHeight = Math.max(...tracks.map(d => d.height!));
```

Comment:
can we really assert that the width is defined? This seems dangerous....

```typescript
Math.max(1, 2, 3, undefined, 5, 6);
// NaN
```

Reply:
It would be safer to filter the tracks by height, or create an object where all tracks have defined sizes.

Reply:
At a minimum, this assertion 

```typescript
        const tracks = spec.tracks as (Omit<Track, 'width' | 'height'> & { width: number, height: number})[];
```